code/modules/mob/mob.dm

```diff
@@ -852,31 +852,40 @@ var/next_mob_id = 0
 /mob/proc/update_health_hud()
 	return
 
-/mob/living/on_varedit(modified_var)
-	switch(modified_var)
+/mob/living/vv_edit_var(var_name, var_value)
+	switch(var_name)
+		if("stat")
+			if((stat == 2) && (var_value < 2))//Bringing the dead back to life
+				dead_mob_list -= src
+				living_mob_list += src
+			if((stat < 2) && (var_value == 2))//Kill he
```

Comment:
where are the stat defines even at so i can fix this to be proper. Its shit code i evicted from https://github.com/tgstation/tgstation/pull/21778/files#diff-8d921f8d3059a9878269cbe94351928bL632 so I didn't even pay attention to it's contents.